packages/commentor/PKGBUILD

```diff
@@ -0,0 +1,29 @@
+pkgname=commentor
+pkgver=19.d81a660
+pkgrel=1
+pkgdesc='Extract all comments from the specified URL resource.'
+groups=('blackarch' 'blackarch-code-audit')
```

Comment:
This don't really do any code audit but only extracts comments, I would rather in `blackarch-misc`